manual/src/main/kotlin/com.r3.corda.finance.manual/types/ManualSettlement.kt

```diff
@@ -0,0 +1,20 @@
+package com.r3.corda.finance.manual.types
+
+import com.r3.corda.finance.manual.flows.MakeManualPayment
+import com.r3.corda.finance.obligation.types.OffLedgerPayment
+import net.corda.core.identity.Party
+
+/**
+ * Terms specific to settling manually.
+ */
+data class ManualSettlement(
+        override val accountToPay: String,
+        val remittanceInformation : String
+) : OffLedgerPayment<MakeManualPayment<*>> {
+    override val settlementOracle: Party? = null
+    override val paymentFlow: Class<MakeManualPayment<*>> = MakeManualPayment::class.java
+
+    override fun toString(): String {
+        return "Pay manually $accountToPay and use $settlementOracle as settlement Oracle."
```

Comment:
Presumably there is no Oracle if you are using manual payment?

Reply:
Yes, thank you. Will change